src/vma/proto/mem_buf_desc.h

```diff
@@ -103,12 +103,7 @@ class mem_buf_desc_t {
 		int8_t		n_frags;	//number of fragments
 		bool 		is_vma_thr; 	// specify whether packet drained from VMA internal thread or from user app thread
 		bool		is_sw_csum_need; // specify if software checksum is need for this packet
-
-#ifdef DEFINED_VMAPOLL
 		bool 		vma_polled;
-#else
-		bool		pad[1];
```

Comment:
while I still think it is not needed I had restored #ifdef VMAPOLL will the using new POLL CQ by VMAPOLL.